src/util/map-styles.js

```diff
@@ -93,6 +100,6 @@ export const getTextStyles = () => {
 export const getOverlapHintStyles = (radius) => [
   {
     filter: ['has', 'overlapHint'],
-    style: styleCircle(radius)
+    style: styleCircle(radius, 0.51)
```

Comment:
This is only shown in development. There's a toggle for it in DevTools. I just eyeballed 0.51. I might change it later.